libs/domain/content/src/services/adapter/content.adapter.ts

```diff
@@ -1,16 +1,25 @@
 import { Observable } from 'rxjs';
-import { Content, ContentQualifier } from '../models';
+import { Content, ContentQualifier } from '../../models';
+
+export const ContentAdapter = 'oryx.ContentAdapter*';
+export const ContentConfig = 'oryx.ContentConfig*';
 
 export interface ContentAdapter {
+  getName(): string;
```

Comment:
wondering about eithe using `getName` or named multi token, to not pollute the service? 🤔